modules/behavioural_qc/php/models/behaviouraldto.class.inc

```diff
@@ -19,6 +19,97 @@ namespace LORIS\behavioural_qc\Models;
  */
 class BehaviouralDTO implements \LORIS\Data\DataInstance
 {
+    /**
+     * The instrument
+     *
+     * @var string
+     */
+    public $instrument = '';
```

Comment:
visibility should be `private`
PDO::FETCH_CLASS can populate private class variables.